buildscripts/scripts/python.sh

```diff
@@ -33,9 +33,9 @@ recompile_py () {
 prune_stdlib () {
 	local delete=(
 		pydoc_data turtledemo # docs
-		test unittest/test # unittests
+		test # unittests
```

Comment:
This line can be omitted if the new build flag to disable building tests is used.